ocean/helium/snapshots.py

```diff
@@ -131,10 +136,267 @@ def download_file_from_snapshot(src, dst, snapshothash):
     snapshot_data = read_snapshot_by_hash(bucket, snapshothash)
     obj_rec = snapshot_data['contents'][key]
     return download_file(src, dst, version=obj_rec['VersionId'])
-    
+
 
 def download_bytes_from_snapshot(src, snapshothash):
     bucket, key = split_path(src)
     snapshot_data = read_snapshot_by_hash(bucket, snapshothash)
     obj_rec = snapshot_data['contents'][key]
     return download_bytes(src, version=obj_rec['VersionId'])
+
+def hash_file(path):
+    """ Returns SHA256 hash of file at path. """
+    with open(path, 'rb') as f:
+        return hashlib.sha256(f.read()).hexdigest()
+
+class PackageException(Exception):
```

Comment:
Let's move this into an `exceptions.py` module before landing.

Reply:
OK